### PR TITLE
Konstantinos/ To develop a new popup message to appear for Non EU IP user in eu.deriv.com

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_non-eu-redirect-popup.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    flex-grow: 0;
+`
+const RedirectNonEuLogin = () => {
+    window.location.href = `https://${deriv_com_url}`
+}
+const NonEuRedirectPopUp = ({ is_open, setShowNonEuPopup }: PopupModalTypes) => {
+    const toggle = () => setShowNonEuPopup(!is_open)
+    return (
+        is_open && (
+            <>
```
Comment (prince-deriv) [inline src/components/custom/_non-eu-redirect-popup.tsx:89]: Do we need this?

File: src/components/layout/nav/components/right-section.tsx [hunk 2]
```diff
@@ -40,9 +45,11 @@ const RightSection = ({
     hide_language_switcher,
     hide_signup_login,
 }: RightSectionProps) => {
+    const { non_eu_popup } = React.useContext(DerivStore)
+    const [_, setShowNonEuPopup] = non_eu_popup
```
Comment (prince-deriv) [inline src/components/layout/nav/components/right-section.tsx:49]: can we try 

```suggestion
    const [, setShowNonEuPopup] = non_eu_popup
```

File: src/components/custom/_non-eu-redirect-popup.tsx (hunk 1 above)
Comment (mohammad-rework) [reply src/components/custom/_non-eu-redirect-popup.tsx:89]: When the user clicks outside of the card we toggle the `is_open` and hide it. Therefor user has a way to cancel the redirection if he wants to stay in eu.deriv.com.

Comment (prince-deriv) [reply src/components/custom/_non-eu-redirect-popup.tsx:89]: I mean the extra fragment `<>`

**Pull request merged**